[DNS] More intro details on different DNS setups (#18526)
cloudflare/cloudflare-docs: src/content/docs/dns/concepts.mdx

---

This page defines and articulates key concepts that are relevant to the Cloudflare DNS service and are used in this documentation. For more concepts and broader descriptions, consider the [Cloudflare Learning Center](https://www.cloudflare.com/learning/dns/what-is-dns/).

## Domain

Also known as domain name, a domain is the string of text that identifies a specific website, such as `google.com` or `facebook.com`. Every time you access a website from your web browser, a DNS query takes place and the domain name is mapped to the actual IP address where the website is [hosted](/fundamentals/setup/manage-domains/).

## Registrar

Before you can start using the Cloudflare DNS service, you must first have a domain. This is achieved by using a service called registrar. As explained in our [Learning Center](https://www.cloudflare.com/learning/dns/glossary/what-is-a-domain-name-registrar/), a registrar handles the reservation of domain names.

Very often the same company that offers domain registration also offers web hosting and DNS management.

> You can register a domain name at coast through [Cloudflare Registrar](/registrar/). Every domain acquired through Cloudflare Registrar must also use Cloudflare as [primary authoritative DNS](#authoritative-dns).

Reviewer: at ~coast~ cost

---

> This page defines and articulates key concepts that are relevant to the Cloudflare DNS service and are used in this documentation. For more concepts and broader descriptions, consider the [Cloudflare Learning Center](https://www.cloudflare.com/learning/dns/what-is-dns/).

Reviewer 2: ```suggestion
This page defines and articulates key concepts that are relevant to the Cloudflare DNS service and are used in the Cloudflare DNS documentation. For more concepts and broader descriptions, refer to the [Cloudflare Learning Center](https://www.cloudflare.com/learning/dns/what-is-dns/).
```

## Domain

> Also known as domain name, a domain is the string of text that identifies a specific website, such as `google.com` or `facebook.com`. Every time you access a website from your web browser, a DNS query takes place and the domain name is mapped to the actual IP address where the website is [hosted](/fundamentals/setup/manage-domains/).

Reviewer 2: ```suggestion
Also known as a domain name, a domain is the string of text that identifies a specific website, such as `google.com` or `facebook.com`. Every time you access a website from your web browser, a DNS query runs and the domain name is mapped to the actual IP address where the website is [hosted](/fundamentals/setup/manage-domains/).
```
Second sentence is in passive voice. Would be worth changing it to "the DNS service maps the domain name is mapped to the actual IP addres where..."

## Registrar

Before you can start using the Cloudflare DNS service, you must first have a domain. This is achieved by using a service called registrar. As explained in our [Learning Center](https://www.cloudflare.com/learning/dns/glossary/what-is-a-domain-name-registrar/), a registrar handles the reservation of domain names.

Very often the same company that offers domain registration also offers web hosting and DNS management.

> You can register a domain name at cost through [Cloudflare Registrar](/registrar/). Every domain acquired through Cloudflare Registrar must also use Cloudflare as [primary authoritative DNS](#authoritative-dns).

Reviewer 2: ```suggestion
You can register a domain name at cost through [Cloudflare Registrar](/registrar/). Every domain acquired through Cloudflare Registrar must also use Cloudflare as their [primary authoritative DNS](#authoritative-dns).
```

Refer to [Nameservers](/dns/nameservers/) for details on the different nameserver offerings.

## Authoritative DNS

Authoritative DNS refers to the service whose nameservers provide the final information mapping a hostname (such as `example.com` or `blog.example.com`) to the IP address that hosts the corresponding content or resources.

This is important because the performance of such authoritative DNS services determine how available, resilient, and performant your website or application is. Cloudflare DNS is an authoritative DNS service leveraging Cloudflare's global network. Refer to [How Cloudflare works](/fundamentals/concepts/how-cloudflare-works/) for details.

## DNS setups

It is also possible that one same company will use more than one DNS provider. Usually, this relates to making a domain more resilient - if one provider faces an outage, the nameservers operated by the other DNS provider will most likely still be available.

> In this context, you can have a primary DNS setup, when you use Cloudflare to manage your DNS records, or a [secondary DNS setup](/dns/zone-setups/zone-transfers/cloudflare-as-secondary/), when your DNS records are managed on a different provider and Cloudflare simply receives zone transfers containing your DNS records.

Reviewer 3: should we mention directly here that Secondary is only for Enterprise?
I feel some users would love to use Cloudflare as Secondary: it would set the expectations straight from the begining

When you have a primary DNS setup, you can either use only Cloudflare (also known as [Full setup](/dns/zone-setups/full-setup/)), or you can use Cloudflare and another provider, where the other provider is the one to receive [outgoing zone transfers](/dns/zone-setups/zone-transfers/cloudflare-as-primary/) from Cloudflare.

Finally, as Cloudflare also works as a [reverse proxy](/fundamentals/concepts/how-cloudflare-works/#how-cloudflare-works-as-a-reverse-proxy), [partial (CNAME) setups](/dns/zone-setups/partial-setup/) can be used when you do not want Cloudflare to be [authoritative](#authoritative-dns) for your domain but you still want to proxy individual subdomains through Cloudflare.

## Zone

> DNS zone is an administrative concept used for delegating control over a given domain and its subdomains. You can read more about this in the [specific Learning Center article](https://www.cloudflare.com/learning/dns/glossary/dns-zone/).

Reviewer 2: ```suggestion
DNS zone is an administrative concept used for delegating control over a given domain and its subdomains. Read more about DNS zones in the ["What is a DNS zone?" Learning Center article](https://www.cloudflare.com/learning/dns/glossary/dns-zone/).
```
